sbol3/document.py

```diff
@@ -5,6 +5,7 @@
 import os
 import posixpath
 import warnings
+from pathlib import Path
```

Comment:
I generally would have preferred `import pathlib`, and then `pathlib.Path` in the code. It's a minor thing.